action.yml

```diff
@@ -24,5 +24,22 @@ inputs:
     default: main
 
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
+  using: "composite"
+  steps:
+    - name: Set up pixi
+      uses: prefix-dev/setup-pixi@ba3bb36eb2066252b2363392b7739741bb777659  # v0.8.1
+      with:
```

Comment:
Should we pin the pixi version? I think it downloads the latest pixi if there is no version specified

Reply:
it downloads the latest, yes. as there is no proper autoupdate mechanism i personally would keep it at `latest` or suggest you write your own. otherwise you will end up with stale pixi versions in ci

Reply:
> I personally might soften my view here and say that having the latest pixi get installed is probably a good idea / will have no effect given that the lock file is the thing of importance.

I am fine with that too :)